community_tutorials_and_guides/blazingsql/bsql_vs_pyspark_netflow.ipynb

```diff
@@ -41,12 +41,12 @@
     }
    ],
    "source": [
-    "import sys \n",
-    "# point import path notebooks-contrib/utils\n",
-    "sys.path.append('../../../utils/')\n",
-    "from sql_check import bsql_start\n",
-    "# check that BlazingSQL is installed\n",
-    "bsql_start()"
+    "# import sys \n",
```

Comment:
if these are not required then i believe this cell can be removed

Reply:
@Nicholas-7 these just need to be uncommented.  @Salonijain27 , user *may* need them if they didn't install RAPIDS+BlazingSQLg

Reply:
I was reviewing the changes and not the whole code, hence, i said if its not required it should be removed 😄

Reply:
haha, i know.  Just putting in an opinion and thought :)